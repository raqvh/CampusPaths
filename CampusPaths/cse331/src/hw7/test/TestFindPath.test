# Test that MarvelPaths2 uses Dijkstra to find the min-cost path

LoadGraph csefriends CSEfriends.tsv

# Simple case
FindPath csefriends Jack Duncan

# Different case
FindPath csefriends Ashley Duncan
