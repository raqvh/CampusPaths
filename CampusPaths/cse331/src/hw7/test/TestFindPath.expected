# Test that MarvelPaths2 uses Dijkstra to find the min-cost path

loaded graph csefriends

# Simple case
path from Jack to Duncan:
Jack to Duncan with weight 1.000
total cost: 1.000

# Different case
path from Ashley to Duncan:
Ashley to Cat with weight 1.000
Cat to Novin with weight 1.000
Novin to Duncan with weight 1.000
total cost: 3.000
